### PR TITLE
Fix `Second` parsing in the marlowe-cli

### PR DESCRIPTION
**Pull request by paluh**
I've incorporated a minor clean up into this PR (empty module removal - `Test/Generator.hs`).

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
        - Review required
        - [ ] Substantial changes to code, test, or documentation
        - [ ] Change made to Marlowe validator (@bwbush and @palas must be included as reviewers)
        - Review not required
        - [ ] Minor changes to non-critical code, documentation, nix derivations, configuration files, or scripts
        - [ ] Formatting, spelling, grammar, or reorganization
    - [x] Reviewer requested

### CODE REVIEW

**Review by bwbush — APPROVED** [review]
Thanks. I now understand the source of the problem.

**Review by jhbertra — CHANGES_REQUESTED** [review]
This PR introduces a user-facing change (a bugfix), please include a changelog update

**Comment by paluh** [comment]
@bwbush Bumped the "patch" version number (from 0.0.10.2 to 0.0.10.3) - is this proper change?

@jhbertra Added the bug fix changelog.

**Review by jhbertra — CHANGES_REQUESTED** [review]
Please don't update the version yet - this happens when we release

File: marlowe-cli/marlowe-cli.cabal [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name: marlowe-cli
-version: 0.0.10.2
+version: 0.0.10.3
```
Comment (jhbertra) [inline marlowe-cli/marlowe-cli.cabal:3]:
```suggestion
version: 0.0.10.2
```

**Pull request merged**